bundles/org.openhab.binding.sonyprojector/src/main/java/org/openhab/binding/sonyprojector/internal/communication/SonyProjectorCalibrationPreset.java

```diff
@@ -144,6 +157,27 @@ public static List<StateOption> getStateOptions(int category) {
         return options;
     }
 
+    /**
+     * Get the list of {@link CommandOption} associated to the available calibration presets for a particular category
+     * of
+     * projector models
```

Comment:
Formatting, very minor:
```suggestion
     * of projector models
```